Mathlib/Data/BitVec/Defs.lean

```diff
@@ -1,146 +0,0 @@
-/-
-Copyright (c) 2015 Joe Hendrix. All rights reserved.
-Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Joe Hendrix, Sebastian Ullrich, Harun Khan, Alex Keizer, Abdalrhman M Mohamed
--/
-import Mathlib.Algebra.Group.Fin
-import Mathlib.Data.Nat.Bits
-import Mathlib.Data.ZMod.Defs
-import Batteries.Data.BitVec
-
-#align_import data.bitvec.core from "leanprover-community/mathlib"@"1126441d6bccf98c81214a0780c73d499f6721fe"
-
-/-!
-# Basic operations on bitvectors
-
-Std has defined bitvector of length `w` as `Fin (2^w)`.
-Here we define a few more operations on these bitvectors
-
-## Main definitions
-
-* `Std.BitVec.sgt`: Signed greater-than comparison of bitvectors
-* `Std.BitVec.sge`: Signed greater-equals comparison of bitvectors
-* `Std.BitVec.ugt`: Unsigned greater-than comparison of bitvectors
-* `Std.BitVec.uge`: Unsigned greater-equals comparison of bitvectors
-
--/
-
-variable {w v : Nat}
-
-namespace BitVec
-
-#align bitvec BitVec
-#align bitvec.zero BitVec.zero
-
-/-!
-## Constants
--/
-
-/-- The bitvector representing `1`.
-    That is, the bitvector with least-significant bit `1` and all other bits `0` -/
-@[simp] abbrev one (w : ℕ) : BitVec w := 1
-#align bitvec.one BitVec.one
-
-#align bitvec.cong BitVec.cast
-#align bitvec.append BitVec.append
-#align bitvec.shl BitVec.shiftLeft
-#align bitvec.ushr BitVec.ushiftRight
-#align bitvec.sshr BitVec.sshiftRight
```

Comment:
Repeating a comment that was made privately; we should keep the `#align`s around, at least as long as we keep any `align`s around.

Reply:
I've restored the #aligns, turning them into #noalign when something is being deleted.

Reply:
Thanks!